noma/config.py

```diff
@@ -28,6 +28,14 @@
 HOME_PATH = Path.home()
 COMPOSE_MODE_PATH = NOMA_SOURCE / "compose" / LND_MODE
 
+"""Bitcoind Paths"""
+BITCOIN_PATH = NOMA_SOURCE / "bitcoind"
```

Comment:
This notation looks like confusing division IMO, any way you can turn it into a string with variables within?

Reply:
This `pathlib` notation is part of Python since 3.4.

In the unlikely event that someone passes an integer instead of a string or valid path object the interpreter would raise an error immediately.

The alternatives would be harder to read and more verbose. Nonetheless, I'm willing to revisit the config topic at a later date. Perhaps a more user-friendly `toml` file can be exposed while this one is made private.

Reply:
It's not utterly confusing.  It is possible to understand it, just initially confusing to python n00bs.

Having said that, it's not worth fighting against the language.  Let's leave it how it is.  It's well commented, variable names are clear, I'm sure readers can deal with the funky notation :).